.github/ansible/scripts/init_pageserver.sh

```diff
@@ -18,7 +20,9 @@ cat <<EOF | tee /tmp/payload
   "http_host": "${HOST}",
   "http_port": 9898,
   "active": false,
-  "availability_zone_id": "${AZ_ID}"
+  "availability_zone_id": "${AZ_ID}",
+  "disk_size": ${DISK_SIZE},
```

Comment:
NIT: what about int overflow if the disk size is enormous? Would you be able to parse it correctly in all the cases?

Reply:
It is int64 in control plane, more than enough for all real disk sizes